management-account/terraform/iam-users.tf

```diff
@@ -81,6 +81,26 @@ resource "aws_iam_user_group_membership" "jason_birchall" {
   ]
 }
 
+##################
+# Julie Robertson #
+##################
+resource "aws_iam_user" "julie_robertson" {
+  name          = "JulieRobertson"
+  path          = "/"
+  force_destroy = true
+  tags          = {}
+}
+
+# User membership
+resource "aws_iam_user_group_membership" "julie_robertson" {
+  user = aws_iam_user.jason_birchall.name
```

Comment:
You'll want to change this to `aws_iam_user.julie_robertson.name`.

Reply:
Whoops! Fixing…